src/api/account_api.js

```diff
@@ -457,6 +457,9 @@ module.exports = {
                     },
                     endpoint_type: {
                         $ref: 'common_api#/definitions/endpoint_type'
+                    },
+                    ignore_name_already_exist: {
+                        type: 'boolean'
```

Comment:
Are we calling it?

Reply:
In the comment above, I added a link to a PR in the operator that will use that field.

Reply:
👍